deploy/nginx/default.conf.template

```diff
@@ -41,4 +42,9 @@ server {
   location / {
     proxy_pass ${MODOS_LOCAL_URL} ;
   }
+  # terminology code matching server
+  location /fuzon {
+    rewrite ^/fuzon/(.*) /$1 break ;
+    proxy_pass ${FUZON_LOCAL_URL} ;
```

Comment:
**question(clarification)**: the config template provides both a public & local setup. How is this config used exactly to make sure that we're dealing with either public or local setup? The main reason I'm asking is if it makes more sense to separate the two into different files or make it clearer what needs to be modified in which case.

Reply:
Documented in [06ca00b](https://github.com/sdsc-ordes/modos-api/pull/106/commits/06ca00bade6c6e0a11f79f786ea93293c48651f0) @marftn  do you think this whole thing makes sense, or is this some kind of antipattern?